### ISSUE TITLE
Unable Move window by keyboard only (up/down/left/right) ?

### ISSUE THREAD

**Issue by tankb52** [labwc/labwc#2551]
### Problem description

I add an "Move" action to  menu.xml.  
If i select it by mouse, it doesn't work.
If i select it by keyboard, it worked , but the window only follow the movement of the mouse cursor. 

Can I use the arrow keys  (up/down/left/right)  on my keyboard to move windows?

### Steps to reproduce

1. add "Move" action to menu.xml 
2. reconfigure labwc.
3. Open a window like thunar or gvim.
4. "Alt + Space"  to open the menu.    
    4.1  click the "Move" item by mouse.
    4.2 Move the mouse.
5. "Alt + Space" to open the menu.
    5.1 Select the "Move" item by keyboard arrow keys.
    5.2 Press "Enter" to activate the menu item.
    5.3 Press  the arrow keys  (up/down/left/right)
    5.4 Move the Mouse cursor.
6. Press the "ESC" or "Enter" key to  leave or confirm the moving stance.
7. Click the left mouse button to confirm the movement.

--------------------
4.2  Unable to move the window.  The cursor of the mouse does not change to a movement style.

5.3 Unable to move the window.  But the cursor of the mouse change to a movement style.
5.4 Ability to move the window with the mouse.

6. Unable to  leave or confirm the moving stance by keyboard. 
7. Ability to confirm results with the mouse.

### labwc build source

Release

### labwc version

0.8.2

### labwc environment

Nested on Wayland

### Distribution

Debian  trixie

**Comment by tankb52** [comment]
This my menu.xml  content.

```xml
```xml
<menu id="client-menu">
 ......
  <item label="Move">
    <action name="Move" />
  </item>

``` 
```

**Comment by Narrat** [comment]
Just curious: Why via an option from the menu and not directly with keybinds?

**Comment by tankb52** [comment]
> Just curious: Why via an option from the menu and not directly with keybinds?

When I used Openbox before, Some software's window border outside the screen after it  running, then they need to be moved. 
However, the probability of this happening is very low, so I don't want to waste a shortcut key.

**Comment by johanmalm** [comment]
It’s just not implemented yet. This should be a feature request.
